### ISSUE TITLE
Expand JSON-LD on response

### ISSUE THREAD

**Issue by umbreak** [BlueBrain/nexus-kg#98]
When retrieving a resource, we would like to add the possibility to represent it as an expanded JSON-LD.

**Comment by umbreak** [comment]
We were discussing yesterday how to represent that in terms of API. We thought about having a query param called `output` or `format` which has the following option

- default
- jsonld-expand
- jsonld-compact
- json

and it will be used like `...?format=jsonld-expand`

**Comment by MFSY** [comment]
I like ?format=....
But you may want to look at Jena json-ld rdf formats:
https://jena.apache.org/documentation/io/rdf-output.html#json-ld

Json-ld is serialized in Jena using different flavors: "compacted", "expanded", "flattened", "framed"

**Comment by umbreak** [comment]
Yes you are right. In the POC I did I used the Jena [JsonLDWriter](https://jena.apache.org/documentation/javadoc/arq/org/apache/jena/riot/writer/JsonLDWriter.html) which allows you to specify those types. So then we could have:

- compacted
- expanded
- flattened
- framed
- default (without formatting, as the customer pushed it)
